opensourceleg/messaging/grpc.py

```diff
@@ -0,0 +1,163 @@
+import importlib
+import math
+import os
+import time
+from concurrent import futures
+
+from opensourceleg.logging import LOGGER
+
+try:
+    import grpc
+    from google.protobuf import descriptor_pb2
+    from google.protobuf.compiler import plugin_pb2
+    from grpc_tools import protoc
+
+except ImportError:
+    LOGGER.error("grpcio-tools, types-protobuf, and grpcio must be installed.")
+
+
+class GRPCManager:
```

Comment:
Is there a better name for this class that speaks more to its function rather than how it achieves that function? Perhaps something like GUI manager? Or would this be used outside a GUI context?

Reply:
Yes the GRPCManager class can be used outside the GUI context and @VarunSatyadevShetty is planning to add more features to this module in the near future.

Reply:
The name **GPRCManager** I think can be somewhat general across various messaging libraries, like ZMQManager, etc

But I'm also open to suggestions! @VarunSatyadevShetty Please feel free to chime in!

Reply:
Ok sounds good - I'll trust you guys!